firmware/integration/fome_config.txt

```diff
@@ -1582,6 +1580,9 @@ uint16_t[SCRIPT_TABLE_8] vvtTable2RpmBins;;"RPM", 1, 0, 0, 18000, 0
 int16_t[IGN_RPM_COUNT x IGN_LOAD_COUNT] autoscale ignitionTable;;"deg", 0.1, 0, -20, 90, 1
 uint16_t[IGN_LOAD_COUNT] ignitionLoadBins;;"Load", 1, 0, 0, 1000, 0
 uint16_t[IGN_RPM_COUNT] ignitionRpmBins;;"RPM",	   1, 0, 0, 18000, 0
+int16_t[IGN_RPM_COUNT x IGN_LOAD_COUNT] autoscale trailingIgnitionTable;;"deg", 0.1, 0, -20, 90, 1
```

Comment:
trailing spark angle can't be negative - it's an offset in degrees after the leading spark. This can probable be `uint8_t autoscale .... "deg", 0.1, 0, 0, 25, 1`